.storybook/main.js

```diff
@@ -1,4 +1,11 @@
 module.exports = {
```

Comment:
I've never seen Storybook used before.  I'm glad you chose to use it. I will be hopefully using your use of it as a guideline to learn myself. Thank you.